spec/shipment_spec.rb

```diff
@@ -268,26 +253,14 @@
       shipment_data = Fixture.one_call_buy_shipment
       shipment_data[:insurance] = 0
 
-      shipment = described_class.create(shipment_data)
+      shipment = client.shipment.create(shipment_data)
 
-      shipment.insure(
+      insured_shipment = client.shipment.insure(
+        shipment.id,
         amount: '100',
       )
 
-      expect(shipment.insurance).to eq('100.00')
-    end
-
-    it 'insures a shipment with unwrapped params' do
-      # If the shipment was purchased with a USPS rate, it must have had its insurance set to `0` when bought
-      # so that USPS doesn't automatically insure it so we could manually insure it here.
-      shipment_data = Fixture.one_call_buy_shipment
-      shipment_data[:insurance] = 0
-
-      shipment = described_class.create(shipment_data)
-
-      shipment.insure(100)
-
-      expect(shipment.insurance).to eq('100.00')
```

Comment:
issue: removing tests isn't what we're after. If there is a test, it had a purpose (either code coverage or to protect against a regression we've faced in the past), removing it removes that protection.

Reply:
I realized this test is almost the same as the above test with different types of string and int of amount. Added them back since Ruby is not a type-strict language